franklin/builder/views.py

```diff
@@ -1,16 +1,36 @@
 import logging
 
-from rest_framework.status import HTTP_200_OK
+from django.shortcuts import get_object_or_404
+
+from rest_framework.decorators import api_view, permission_classes
 from rest_framework.exceptions import NotFound, ParseError
 from rest_framework.permissions import AllowAny
 from rest_framework.views import APIView
 from rest_framework.response import Response
+from rest_framework.status import HTTP_200_OK
 
 from .models import Build, BranchBuild, Deploy, Environment, Site
+from .serializers import BuildSerializer
+from core.exceptions import BadRequest
 
 logger = logging.getLogger(__name__)
 
 
+@api_view(['GET'])
+@permission_classes((AllowAny, ))
+def domain(request):
+    """
+    Returns details about a domain managed by Franklin
+    """
+    if request.method == 'GET':
+        domain = request.GET.get('domain', False)
```

Comment:
You can remove the default of `False`. `get()` returns `None` if the key is not found so you can just rely on that default behavior.